README.md

```diff
@@ -1,6 +1,6 @@
-# `sgeop`: Street Geometry Processing Toolkit
+# `neatnet`: Street Geometry Processing Toolkit
```

Comment:
Still need to decide on a nifty one-liner, but that can wait for now.

Reply:
Yeah, we don't need it to rename the stuff.,